controls/cis_rhel8.yml

```diff
@@ -1344,191 +1277,48 @@ controls:
-    status: manual
-
-  - id: 3.4.3.2.3
-    title: Ensure iptables rules exist for all open ports (Automated)
-    levels:
-      - l1_server
-      - l1_workstation
-    status: partial
-    notes: |-
-      Currently the check is only available in SCE and an automated remediation is not expected.
-    rules:
-      - iptables_rules_for_open_ports
-
-  - id: 3.4.3.2.4
-    title: Ensure iptables default deny firewall policy (Automated)
-    levels:
-      - l1_server
-      - l1_workstation
-    status: automated
-    related_rules:
-      - set_iptables_default_rule
-
-  # NEEDS RULE
-  - id: 3.4.3.2.5
-    title: Ensure iptables rules are saved (Automated)
-    levels:
-      - l1_server
-      - l1_workstation
-    status: pending
-
-  - id: 3.4.3.2.6
-    title: Ensure iptables is enabled and active (Automated)
-    levels:
-      - l1_server
-      - l1_workstation
-    status: automated
+      is in use. When using firewalld the base chains are installed by default.
     related_rules:
-      - service_iptables_enabled
-
-  # NEEDS RULE
-  # https://github.com/ComplianceAsCode/content/issues/5258
-  - id: 3.4.3.3.1
-    title: Ensure ip6tables loopback traffic is configured (Automated)
-    levels:
-      - l1_server
-      - l1_workstation
-    status: pending
-
-  - id: 3.4.3.3.2
-    title: Ensure ip6tables outbound and established connections are configured (Manual)
-    levels:
-      - l1_server
-      - l1_workstation
-    status: manual
+    - set_nftables_new_connections
 
-  - id: 3.4.3.3.3
-    title: Ensure ip6tables firewall rules exist for all open ports (Automated)
+  - id: 3.4.2.5
+    title: Ensure nftables default deny firewall policy (Automated)
     levels:
       - l1_server
       - l1_workstation
-    status: partial
+    status: supported
     notes: |-
-      Currently the check is only available in SCE and an automated remediation is not expected.
-    rules:
-      - ip6tables_rules_for_open_ports
-
-  - id: 3.4.3.3.4
-    title: Ensure ip6tables default deny firewall policy (Automated)
-    levels:
-      - l1_server
-      - l1_workstation
-    status: automated
-    rules:
-      - set_ip6tables_default_rule
-
-  # NEEDS RULE
-  - id: 3.4.3.3.5
-    title: Ensure ip6tables rules are saved (Automated
-    levels:
-      - l1_server
-      - l1_workstation
-    status: pending
-
-  # NEEDS RULE
-  - id: 3.4.3.3.6
-    title: Ensure ip6tables is enabled and active (Automated)
-    levels:
-      - l1_server
-      - l1_workstation
-    status: pending
+      RHEL systems use firewalld for firewall management. Although nftables is the default
+      back-end for firewalld, it is not recommended to use nftables directly when firewalld
+      is in use.
     related_rules:
-      - service_ip6tables_enabled
-      # This rule returns error in RHEL8 because the ip6tables service is not there.
-      # This requirement and respective rules should be reviewed.
+      - nftables_ensure_default_deny_policy
```

Comment:
It would be good to include the cis@rhel8 references on these related_rules too.